src/Stratis.Bitcoin/Consensus/Rules/RuleContext.cs

```diff
@@ -70,12 +47,7 @@ public RuleContext(ValidationContext validationContext, NBitcoin.Consensus conse
             this.ConsensusTip = consensusTip;
             this.PreviousHeight = consensusTip.Height;
```

Comment:
Like here the naming is weird.

Reply:
Its sort of the same thing, but I can change I dont mind actually